packages/rules-external-tools/src/tests/eslint-prefer-const-rule.test.ts

```diff
@@ -2,22 +2,21 @@ import * as assert from "assert";
 import baretest from "baretest";
 import { inFixtureDir } from "./test-helper";
 import { ESLintPreferConstRule } from "../eslint-prefer-const-rule";
-import { ESLintRules } from "../eslint-config";
 import { asBollDirectory, getSourceFile, NullLogger, Package, ResultStatus } from "@boll/core";
 
 export const test: any = baretest("ESLint Prefer Const");
 
-test("Should succeed because prefer-const rule is set to 'error'", () => {
-  inFixtureDir("configs/eslint/prefer-const", async () => {
+test("Should succeed because prefer-const rule is set to 'error'", async () => {
+  await inFixtureDir("configs/eslint/prefer-const", async () => {
```

Comment:
Good catch, I noticed this was introducing race conditions and had a fix out in a development branch I'm working in, so thanks for fixing!